docs/tutorial/image_det.md

```diff
@@ -1,3 +1,169 @@
 # Training for Object Detection
 
-Object detection is in preparation.
+This guide trains a neural network model to object detection of Human Face and Human Hand in the [Open Images Dataset V4](https://storage.googleapis.com/openimages/web/index.html) on GPU server.
+
+<img src="../_static/openimages_v4.png" width="600">
+
+## Preparation
+
+The Open Images V4 dataset is available from official website, but original dataset is too large for the tutorial. So, We provide reduced dataset.
+
+Blueoil supports 2 formats for object detection.
+
+- OpenImagev4 format
+- DeLTA-Mark format
+
+Note: *Please see the detail in <a href="../usage/dataset.html">Prepare training dataset</a>*
+
+You can download subset of Open Images V4 from
+[our server](https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/openimages.tgz).
+
+
+	$ wget https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/openimages.tgz
+	$ tar xf openimages.tgz
+
+
+This dataset consists of 5,000 color images in 2 classes, with 2,500 images per class.
```

Comment:
It is friendly to write description what class are include, like this.
```
This dataset consists of 5,000 color images in images in 2 classes `Human face` and `Human hand`, with 2,500 images per class.
```

Don't you public the script to make subset from all open image v4?

Reply:
@ruimashita 
The script need whole of the dataset. It's not useful for tutorial user.
And that script is not ophisticated enough to link...

Reply:
Suggested sentence is fixed.

Reply:
@iizukak 
> The script need whole of the dataset. It's not useful for tutorial user.
And that script is not ophisticated enough to link...

All right, Thanks !